src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -441,6 +448,48 @@ def createlist(self, request):
                     list(row.fields.keys()),
                     list(row.fields.values())
                 )
+
+                try:
+                    # TODO: Make this validation inside the ContriCleaner lib.
+                    item.full_clean(exclude=(
```

Comment:
@vladsha-dev why not to make this as part of this PR?

Reply:
Will do as part of this PR.

Reply:
I don't think we need to add it to the ContriCleaner library for now, as this check wasn't performed for the API requests previously. However, I believe we need to address it for list uploads, as it was already functioning for them. Afterward, I will create a follow-up Jira ticket to consolidate constraints and validate latitude and longitude. The intent of the OSDEV-966 Jira task was to ensure backward compatibility, not to introduce new functionality.

Reply:
Hi @killev,
After discussing with Mariel and Vanessa, we agreed to preserve the validation as it currently functions on production for list uploads. Since the goal of the OSDEV-966 ticket is to revert to the current production state, particularly for list uploads, I propose creating a separate ticket for your idea. While your suggestion may streamline API uploads and list uploads validation, it would necessitate rewriting and reestimating the OSDEV-966 ticket.
Does it sound good to you?

Reply:
Will do.